FLEDGE.md

```diff
@@ -155,9 +155,19 @@ The `executionMode` attribute is optional, and may contain one of the following
    mode does not have the same limitations on what top-level sites can join or leave
    the interest group.
 
-The `ads` list contains the various ads that the interest group might show.  Each entry is an object that includes both a rendering URL and arbitrary metadata that can be used at bidding time.
+The `ads` list contains the various ads that the interest group might show.  Each entry is an object that must contain an `renderUrl` property with the URL for the ad that would be shown. An ad may also have the following optional properties:
```

Comment:
>An ad may also have the following optional properties:
Wouldn't at least one `sizeGroup` be needed (it is part of the k-anon computation), else how would the auction determine an ad's suitability? Or is it permissible to specify an `adSize` on an ad?

Reply:
Sorry I muddled two things there @brusshamilton.  My 

>how would the auction determine an ad's suitability

intended to ask 

how would the auction determine the ad's suitability to fit the ad slot dimensions?

Reply:
Presumably the bidding script has some way of knowing. Either through the metadata associated with an ad or other data in the interest group.

We are going to be adding the sizeGroup eventually (see pull request #417), but it doesn't make sense to ad a reference to it before that pull request lands.